gui/src/components/health/cortx-health-graphical.vue

```diff
@@ -0,0 +1,544 @@
+/*
+* CORTX-CSM: CORTX Management web and CLI interface.
+* Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Affero General Public License as published
+* by the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Affero General Public License for more details.
+* You should have received a copy of the GNU Affero General Public License
+* along with this program. If not, see <https://www.gnu.org/licenses/>.
+* For any questions about this software or licensing,
+* please email opensource@seagate.com or cortx-questions@seagate.com.
+*/
+<template>
+  <div>
+    <div id="treeContainer"></div>
+  </div>
+</template>
+<script lang="ts">
+import CortxTabs, { TabsInfo } from "../widgets/cortx-tabs.vue";
+import { Component, Vue, Prop } from "vue-property-decorator";
+import { Api } from "../../services/api";
+import apiRegister from "../../services/api-register";
+import * as d3 from "d3";
+// import { tree } from "vued3tree";
+import * as moment from "moment";
+
+@Component({
+  name: "cortx-health-graphical"
+})
+export default class CortxHealthGraphical extends Vue {
+  public healthQueryParams: any = {};
```

Comment:
Is this variable required to be declared global? If not please declare it locally.

Reply:
removed variable from global scope and added it locally